include/envoy/upstream/host_description.h

```diff
@@ -28,7 +28,6 @@ namespace Upstream {
   COUNTER(rq_timeout)                                                                              \
   COUNTER(rq_success)                                                                              \
   COUNTER(rq_error)                                                                                \
-  COUNTER(rq_dropped)                                                                              \
```

Comment:
WDYT about just keeping this and renaming to rq_overflow since it's useful for local debugging and you already wrote the code? It's not a big deal either way. Then for the cluster level dropped you can just increment both places.

Reply:
Outside of the contrived unit test I put together in `router_test`, I don't there is a situation in which `upstream_host_` will be set in the router filter when a `dropped` request is observed. This definitely doesn't happen in maintenance mode, and AFAICT, in overflow scenarios, host picking hasn't taken place yet.

Reply:
Ah yes. I think we used to have an Overflow case for HTTP/2 where we would limit the number of outbound streams, but now we leave that to the codec, which won't actually raise the reset as Overflow right now IIRC. Anyway, I agree, fine to remove this.

Sorry I forget this in the last review, but can you update docs here: https://envoyproxy.github.io/envoy/operations/admin.html#get--clusters ?

Reply:
Done